db/migrate/20210107161848_remove_external_policies.rb

```diff
@@ -0,0 +1,5 @@
+class RemoveExternalPolicies < ActiveRecord::Migration[5.2]
+  def change
+    ExternalPolicyRemover.run!
```

Comment:
Also a suggestion (from rouge profile removal), to log how many of external profiles it has deleted and if possible for how many accounts. This would be good statistic to report.